### PR TITLE
1542 show term count

### PR DESCRIPTION
**Pull request by ledsoft**
https://kbss.felk.cvut.cz/redmine/issues/1542

Requires https://github.com/kbss-cvut/termit/pull/81 merged and deployed.

### CODE REVIEW

**Comment by ledsoft** [comment]
Backend is deployed, ready for testing.

File: src/i18n/cs.ts [hunk 1]
```diff
@@ -459,6 +459,7 @@ const cs = {
     "term.badge.no-score.tooltip": "Pro tento pojem není skóre k dispozici",
 
     "glossary.title": "Pojmy",
+    "glossary.termCount.tooltip": "Počet pojmů v glosáři",
```
Comment (psiotwo) [inline src/i18n/cs.ts]: What about "Počet pojmů ve slovníku (bez pojmů z importovaných slovníků)"
image

**Review by psiotwo — COMMENTED** [review]
see comment

File: src/i18n/cs.ts [hunk 2]
```diff
@@ -459,7 +459,7 @@ const cs = {
     "term.badge.no-score.tooltip": "Pro tento pojem není skóre k dispozici",
 
     "glossary.title": "Pojmy",
-    "glossary.termCount.tooltip": "Počet pojmů v glosáři",
+    "glossary.termCount.tooltip": "Počet pojmů v glosáři (bez pojmů z importovaných slovníků)",
```
Comment (psiotwo) [inline src/i18n/cs.ts]: I still feel "Počet pojmů v slovníku" is less confusing than "Počet pojmů v glosáři", because the page speaks about vocabularies.

**Review by psiotwo — APPROVED** [review]
lgtm

**Pull request merged**